lib/rubygems/commands/owner_command.rb

```diff
@@ -87,8 +87,8 @@ def manage_owners(method, name, owners)
       begin
         response = send_owner_request(method, name, owner)
 
-        if need_otp? response
-          response = send_owner_request(method, name, owner, true)
+        if need_ask_otp? response
```

Comment:
AFAIR, I had suggested that checking if otp is needed and asking for otp should be separate function. This would cleaner when written like:
```
if need_otp? response
  ask_otp
  response = send_owner_request(method, name, owner)
end
```

Reply:
Yes, I planned to commit this in another commit.